gemrb/core/GUI/GameControl.cpp

```diff
@@ -2079,6 +2075,15 @@ void GameControl::InitFormation(const Point &clickPoint)
 	SetCursor(core->Cursors[IE_CURSOR_USE]);
 }
 
+bool GameControl::TryDefaultTalk(Point p) const
+{
+	Actor *targetActor = core->GetGame()->GetCurrentArea()->GetActor(p,
+			target_types & ~(GA_NO_HIDDEN /*| GA_NO_ALLY | GA_NO_ENEMY*/ ));
```

Comment:
Also needs to check GA_NO_DEAD and GA_NO_UNSCHEDULED. The other two can go.

Reply:
can go you mean i need to delete the comments or uncomment them?

Reply:
Yeah, delete the commented stuff.